content/10.diagnostics.md

```diff
@@ -81,11 +104,22 @@ The authors are now working to get the assay into clinical use [@url:https://www
 
 Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
 Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
+Due to the long incubation times and delayed immune responses of infected patients, serological tests are too insensitive for a diagnosis in the early stages of an infection.
+The limitation on timing makes serological tests vfar less useful for enabling test-and-trace strategies.
 
 ### Possible Alternatives to Current Practices for Identifying Active Cases
 
-- Are there other approaches that have worked for diagnosing other viruses at a rapid pace in large numbers of people?
-- What are some approaches people are currently testing for detecting live viruses, especially SARS-CoV-2?
+Clinical symptoms are too similar to other types of pneumonia to be sufficient/reliable as a sole/single diagnostics criterion.
+In addition, as noted above, identifying asymptomatic cases is critical.
+Clinical symptom-based tests are only able to identify symptomatic cases, not presymptomatic or assymptomatic cases.
+They are still important for clinical practice, and for reducing the need for other tests.
+
+X-ray diagnostics have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
+Other studies have shown that specificity varies between radiologists, [@doi:10.1148/radiol.2020200823] though the sensitivity reported here was lower that in the previous paper.
+However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
+X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
+This may still not recognize fully asymptomatic cases.
+In addition, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.
```

Comment:
Why is it inadvisable? Because of the contrast medium used in certain scans? Because of radiation exposure? Who advises against it? Is there a reference?

Reply:
It's not nearly as useful for population level testing because of the cost and time requirements for mass use of chest X-rays.

Reply:
I would explain it as such then.